interchaintest/module_cwhooks_test.go

```diff
@@ -57,6 +57,25 @@ func TestJunoCwHooks(t *testing.T) {
 	require.Equal(t, user.FormattedAddress(), res.Data.DelegatorAddress)
 	require.Equal(t, fmt.Sprintf("%d.000000000000000000", stakeAmt), res.Data.Shares)
 
+	// HIGH GAS TEST
+	// Setup a high gas contract
+	_, contractAddr = helpers.SetupContract(t, ctx, juno, user.KeyName(), "contracts/juno_staking_hooks_high_gas_example.wasm", `{}`)
```

Comment:
make sure to link to this contract source in the README of the interchainttest dir

Reply:
Good catch as well, added the references for the clock testing as well!